src/content/meta/glossary.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Glossary
+description: A page with the term definitions Giant Swarm uses in the documentation.
+search: false
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - "What is a cloud-native developer platform?"
+  - "What is a developer environment?"
+  - "What is a management cluster?"
+  - "What is a managed app?"
+  - "What is the platform API?"
+  - "What is a platform operator?"
+  - "What is smart platform engineering?"
+  - "What is a workload cluster?"
+---
+
+We use a set of terms across our documentation that could be unclear to readers. In the following article we define those terms to clarify their meaning in the context of Giant Swarm.
```

Comment:
[super minor] From a technical writing perspective, I would just throw this sentence away. No introduction is needed, and we also probably shouldn't mention "readers" but rather "you" (you = reader).